### PR TITLE
Cassandra 2.2.2

### PR DESCRIPTION
**Pull request by jond3k**
Upgraded to Cassandra 2.2.2
Also added sha1 since there's no official sha256 on https://cassandra.apache.org/download/

### CODE REVIEW

File: Library/Formula/cassandra.rb [hunk 1]
```diff
@@ -1,9 +1,10 @@
 class Cassandra < Formula
   desc "Eventually consistent, distributed key-value store"
   homepage "https://cassandra.apache.org"
-  url "https://www.apache.org/dyn/closer.cgi?path=/cassandra/2.2.1/apache-cassandra-2.2.1-bin.tar.gz"
-  mirror "https://archive.apache.org/dist/cassandra/2.2.1/apache-cassandra-2.2.1-bin.tar.gz"
-  sha256 "06cad23535333b4dd72596bc9bd0606afd1a1ea74f2a91df571e3e00c51c473f"
+  url "https://www.apache.org/dyn/closer.cgi?path=/cassandra/2.2.2/apache-cassandra-2.2.2-bin.tar.gz"
+  mirror "https://archive.apache.org/dist/cassandra/2.2.2/apache-cassandra-2.2.2-bin.tar.gz"
+  sha1 "1d7e3777d09974386270347bb2d186fa5f89f0b0"
```
Comment (DomT4) [inline Library/Formula/cassandra.rb]: Please remove the `sha1` line. We no longer support the use of anything below SHA256.

Comment (jond3k) [inline Library/Formula/cassandra.rb]: Done. However note that the SHA256 is not currently generated as part of the build process but the SHA1 is.

Comment (DomT4) [inline Library/Formula/cassandra.rb]: Feel free to nag upstream about that. The world really needs to stop relying on SHA1 for cryptographic verification.

**Comment by DomT4** [comment]
Merged in https://github.com/Homebrew/homebrew/commit/04a034b85803bba8cefa13f338e13f465045e61a. Thank you for your contribution to Homebrew @jond3k; we appreciate it! :smiley_cat:

**Pull request closed without merging**